Automattic/wp-calypso
# [WIP] Checklist: Fix broken category prompt in first post tour.

Sometimes steps need to be prepared before they start. In the first post tour, the second step will encourage users to set post categories. However, the description tooltip will be wrong positioned if the sidebar is not open.
This PR will add `wait` prop to the `Step` component. With this function, you can do anything that is necessary to get the step working.

## Test plan

1. Create a new `blog` type website.
2. Move to `/checklist/YOUR_SITE_DOMAIN`.
3. Start the "Publish your first blog post".
4. If the sidebar closed, it should automatically open as the tour progresses.
5. The media dialog will also open automatically if featured image is not set.

cc @markryall @fditrapani
